e2e-tests/genericfile_po.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Page objects to support generic file test cases.
+ */
+
+const { expect } = require('@playwright/test');
+const { url } = require("../fixture");
+const { FileActions } = require('./fileactions.js');
+
+exports.docTypes = ['pad', 'sheet', 'code', 'slide', 'kanban', 'whiteboard', 'form', 'diagram'];
+
+/**
+ * A page object for a CryptPad file.
+ */
+class FilePage {
+
+    constructor(page, testName, mobile) {
+        this.page = page;
+        this.testName = testName;
+        this.mobile = mobile;
+        this.fileActions = new FileActions(page);
+
+        // locators
+        this.mainFrame = page.frameLocator('#sbox-iframe');
+        this.newFile = this.mainFrame.getByRole('menuitem', { name: 'New' }).locator('a');
+        this.storeFile = this.mainFrame.getByRole('menuitem', { name: 'Store' }).locator('a');
```

Comment:
When I run the test against our test branch for the next release `2025.3-test` this "Store" selector is not found. I would understand why, for example it is a `button` and not `a` element, but this part of the code has not changed between the releases so I don't know why it passes on the current version.

Reply:
OK, I'll have a look at this.

Reply:
I don't think I can reproduce this (it would be nice to have a public test instance for 2025.3?). My mental model for now is that this PR works for the cryptpad main branch, and that changes to the main can subsequently lead to a new PR for the test cases here.